# Stops OpenAL Soft from trying to get realtime priority because Emscripten doesn't support all the necessary APIs and this will result in compilation errors when targeting Emscripten.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -565,7 +565,7 @@ if(NOT WIN32)
         set(LINKER_FLAGS ${LINKER_FLAGS} -pthread)
     endif()
 
-    check_symbol_exists(pthread_setschedparam pthread.h HAVE_PTHREAD_SETSCHEDPARAM)
+
 
     # Some systems need pthread_np.h to get pthread_setname_np
     check_include_files("pthread.h;pthread_np.h" HAVE_PTHREAD_NP_H)
